### PR TITLE
Showing domains of accounts

### PR DESCRIPTION
**Pull request by mrsmkl**
Also added GraphQL query.

### CODE REVIEW

File: apps/block_scout_web/lib/block_scout_web/schema/types.ex [hunk 1]
```diff
@@ -167,6 +171,19 @@ defmodule BlockScoutWeb.Schema.Types do
...
+
+    field(:type, :string)
+    field(:element, :string)
+    field(:verified, :boolean)
+  end
+
+  @desc """
+  Celo Claims
+  """
+  object :celo_claims do
```
Comment (mrsmkl) [inline apps/block_scout_web/lib/block_scout_web/schema/types.ex]: Looks like here is some kind of mess up.

File: apps/block_scout_web/lib/block_scout_web/schema.ex [hunk 2]
```diff
@@ -84,6 +84,12 @@ defmodule BlockScoutWeb.Schema do
       resolve(&CeloAccount.get_by/3)
     end
 
+    @desc "Gets all the claims given a address hash."
+    field :celo_claims, list_of(:celo_claims) do
+      arg(:hash, non_null(:address_hash))
+      resolve(&CeloAccount.get_by/3)
```
Comment (mrsmkl) [inline apps/block_scout_web/lib/block_scout_web/schema.ex]: Looks like the problem is here, this call gets directed to wrong clause (line 7 at celo_account.ex).

File: apps/explorer/priv/repo/migrations/20200323090000_create_celo_claims.exs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+  def change do
+    create table(:celo_claims) do
+      add(:address, :bytea, null: false)
+      add(:type, :string, size: 256, null: false)
+      add(:element, :string, size: 2048, null: true)
+      add(:verified, :boolean, null: false)
+      add(:timestamp, :utc_datetime_usec, null: true)
+      add(:inserted_at, :utc_datetime_usec, null: false)
+      add(:updated_at, :utc_datetime_usec, null: false)
```
Comment (mrsmkl) [inline apps/explorer/priv/repo/migrations/20200323090000_create_celo_claims.exs]:
```suggestion
      timestamps(null: false, type: :utc_datetime_usec)
```

Comment (aaitor) [reply apps/explorer/priv/repo/migrations/20200323090000_create_celo_claims.exs]: :+1:

**Pull request merged**